<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mini</class>
 <widget class="QWidget" name="Mini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_switch">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>标准模式</string>
   </property>
  </widget>
  <widget class="RotateCover" name="cover">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>15</y>
     <width>126</width>
     <height>126</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/DISK2_LARGE_TYPE2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pause">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotateCover</class>
   <extends>QLabel</extends>
   <header>rotatecover.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
